### PR TITLE
Use LB as default outbound type

### PR DESCRIPTION
**Pull request by ekarlso**

### CODE REVIEW

**Review by bjorges — CHANGES_REQUESTED** [review]
If you acitvate that block by default all values required for a valid block should have a sane default value as well. To me this seems to be missing. Make sure that usage with all default values used is producing valid and complete resources.

File: variables.tf [hunk 1]
```diff
@@ -111,22 +111,22 @@ variable "load_balancer_sku" {
 }
 
 variable "managed_outbound_ip_count" {
-  default = null
+  default = 1
   type    = number
 }
 
 variable "outbound_ip_address_ids" {
   default = null
```
Comment (bjorges) [inline variables.tf:119]: Should an empty list be the default?

File: variables.tf [hunk 2]
```diff
@@ -111,22 +111,22 @@ variable "load_balancer_sku" {
...
-  type    = list
+  type    = list(any)
 }
 
 variable "outbound_ip_prefix_ids" {
   default = null
```
Comment (bjorges) [inline variables.tf:124]: Should an empty list be the default?

File: variables.tf (hunk 1 above)
Comment (ekarlso) [reply variables.tf:119]: No this is a number / count.

File: variables.tf (hunk 2 above)
Comment (ekarlso) [reply variables.tf:124]: Nope, because if managed ip count then null, they mutually exclude eachother.

File: variables.tf (hunk 1 above)
Comment (bjorges) [reply variables.tf:119]: No, next line defines it as list(any)...

File: variables.tf (hunk 2 above)
Comment (bjorges) [reply variables.tf:124]: Ok, I see. This goes for all of these then and "managed_outbound_ip_count" set to 1 is the default needed to have a valid config.

**Review by bjorges — APPROVED** [review]
LGTM

**Pull request merged**